openshift/template.yaml

```diff
@@ -10,7 +10,7 @@ parameters:
   required: true
 - name: BASE_DNS_DOMAINS # example: name1:id1/provider1,name2:id2/provider2
   value: ''
-- name: OPENSHIFT_INSTALL_RELEASE_IMAGE
+- name: OPENSHIFT_VERSIONS
```

Comment:
you should split it into 2 changes in the saas.yaml
in the first change - add OPENSHIFT_VERSIONS and after your PR will be merged remove the OPENSHIFT_INSTALL_RELEASE_IMAGE from the saas.yaml - otherwise you'll break the integration deployment 
Please open a task for me to add this parameter to production on v1.0.14